ui-particles-angular/projects/ui-particles-angular/src/lib/gio-top-bar/gio-top-bar.component.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@use 'sass:map';
+@use '@angular/material' as mat;
+
+@use '../../scss' as gio;
+
+:host {
+  width: 100%;
+}
+
+.gio-top-bar {
+  display: flex;
+  align-items: center;
+  padding: 16px;
+  background: map.get(gio.$mat-space-palette, lighter10);
+  color: map.get(gio.$mat-dove-palette, default);
+  gap: 18px;
+
+  &__icon {
+    width: 36px;
+    height: 36px;
+    padding: 6px;
+    border-radius: 4px;
+    cursor: pointer;
+  }
+}
+
+.gio-top-bar__icon:hover {
+  background: map.get(gio.$mat-space-palette, lighter20);
```

Comment:
How about adding properly a button ? 
```
<button mat-icon-button class="example-icon" aria-label="Example icon-button with menu icon">
    <mat-icon>menu</mat-icon>
  </button>
```
Play with cursor to simulate a button is not a good practice I think 😁